CMakeLists.txt

```diff
@@ -19,6 +19,8 @@ install(TARGETS "cnpy" LIBRARY DESTINATION lib PERMISSIONS OWNER_READ OWNER_WRIT
 
 if(ENABLE_STATIC)
     add_library(cnpy-static STATIC "cnpy.cpp")
+    target_link_libraries(cnpy-static PUBLIC ${ZLIB_LIBRARIES})
+    target_include_directories(cnpy-static PUBLIC ".")
```

Comment:
Can we remove this line or is it necessary? I can build on my machine w/o it.

Reply:
specifically, the `target_include_directories` command

Reply:
Can you build when including it as a subdirectory?

My CMakeFiles looks like this, more or less...
```
add_subdirectory(external/cnpy)
[...]
add_executable(tests ${SOURCES})
target_link_libraries(tests Catch2::Catch2 cnpy-static gsl::gsl-lite)
```
I may be doing it wrong though :) I clone cnpy as a submodule and include it this way, is there a better approach?